neqo-transport/src/connection.rs

```diff
@@ -466,22 +488,11 @@ impl Connection {
             Some(pkt) => Output::Datagram(pkt),
             None => match self.state {
                 State::Closed(_) => Output::None,
-                _ => match self.next_delay(now) {
-                    Some(delay) => Output::Callback(delay),
-                    None => Output::None,
-                },
+                _ => Output::Callback(self.next_delay(now)),
             },
         }
     }
 
-    /// Process input and generate output.
-    pub fn process(&mut self, dgram: Option<Datagram>, now: Instant) -> Output {
```

Comment:
Why cut this rather than just have this change?

Reply:
I noticed that process() was only used by test code -- actual code needs to have process_input() and process_output() be separate steps with stuff happening in-between.

I'm going to pull this out of this PR and give it its own, I think.